packages/discovery-provider/src/challenges/challenges.dev.json

```diff
@@ -137,5 +137,16 @@
     "starting_block": 0,
     "weekly_pool": 25000,
     "cooldown_days": 7
+  },
+  {
+    "id": "os",
```

Comment:
`o` would be better just because tx size is very very restricted here
https://www.notion.so/audiusproject/Improving-Rewards-Claim-Success-c1827623c85d49dbb438795534ca911b?pvs=4